src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,127 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table = new Node[INITIAL_CAPACITY];
+    private int size;
+
+    private static class Node<K,V> {
```

Comment:
According to the checklist, private methods and classes should be after public ones in your class. Therefore, the Node class should be moved below the public methods.